Sources/Identity/CustomerInfoManager.swift

```diff
@@ -214,9 +233,14 @@ class CustomerInfoManager {
 extension CustomerInfoManager {
 
     @available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, *)
-    func customerInfo(appUserID: String) async throws -> CustomerInfo {
+    func customerInfo(
+        appUserID: String,
+        fetchPolicy: CacheFetchPolicy
+    ) async throws -> CustomerInfo {
         return try await withCheckedThrowingContinuation { continuation in
-            return self.customerInfo(appUserID: appUserID, completion: continuation.resume)
+            return self.customerInfo(appUserID: appUserID,
+                                     fetchPolicy: fetchPolicy,
```

Comment:
I changed all these calls to explicitly pass the previous behavior. But now that we have this feature, it's possible that we might want to customize it and use a different one.